src/main.cpp

```diff
@@ -1031,6 +1031,13 @@ int main ( int argc, char** argv )
     activity.EndActivity();
 #endif
 
+#if defined( _WIN32 )
```

Comment:
Maybe both `activity.EndActivity()` and `::ShowWindow()` blocks should also happen inside the no GUI side of the `catch`...